src/pydocstyle/checker.py

```diff
@@ -38,19 +38,71 @@ class ConventionChecker:
 
     """
 
-    SECTION_NAMES = ['Short Summary',
-                     'Extended Summary',
-                     'Parameters',
-                     'Returns',
-                     'Yields',
-                     'Other Parameters',
-                     'Raises',
-                     'See Also',
-                     'Notes',
-                     'References',
-                     'Examples',
-                     'Attributes',
-                     'Methods']
+    NUMPY_SECTION_NAMES = (
+        'Short Summary',
+        'Extended Summary',
+        'Parameters',
+        'Returns',
+        'Yields',
+        'Other Parameters',
+        'Raises',
+        'See Also',
+        'Notes',
+        'References',
+        'Examples',
+        'Attributes',
+        'Methods'
+    )
+
+    GOOGLE_SECTION_NAMES = (
+        'Args',
+        'Arguments',
+        'Attention',
+        'Attributes',
+        'Caution',
+        'Danger',
+        'Error',
+        'Example',
+        'Examples',
+        'Hint',
+        'Important',
+        'Keyword Args',
+        'Keyword Arguments',
+        'Methods',
+        'Note',
+        'Notes',
+        'Other Parameters',
+        'Parameters',
+        'Return',
+        'Returns',
+        'Raises',
+        'References',
+        'See Also',
+        'Tip',
+        'Todo',
+        'Warning',
+        'Warnings',
+        'Warns',
+        'Yield',
+        'Yields',
+    )
+
+    # Matches anything that fulfills all the following conditions:
```

Comment:
These docs are great. I think these can be formatted better by interleaving these with the actual regex. Something like:

```.py
GOOGLE_ARGS_REGEX = re(
  r'^\s'  # Begins with...
  r'(\w)'  # Followed by...
  ...
)
```

This'll save you from copying the regex to the comments and help the docs stay relevant in case this changes.